cmd/workspace/settings/settings.go

```diff
@@ -18,424 +18,31 @@ var cmdOverrides []func(*cobra.Command)
-	cmd.Long = `Delete the restrict workspace admins setting.
-  
-  Reverts the restrict workspace admins setting status for the workspace. A
-  fresh etag needs to be provided in DELETE requests (as a query parameter). The
-  etag can be retrieved by making a GET request before the DELETE request. If
-  the setting is updated/deleted concurrently, DELETE will fail with 409 and the
-  request will need to be retried by using the fresh etag in the 409 response.`
-
-	cmd.Annotations = make(map[string]string)
-
-	cmd.Args = func(cmd *cobra.Command, args []string) error {
-		check := cobra.ExactArgs(0)
-		return check(cmd, args)
-	}
-
-	cmd.PreRunE = root.MustWorkspaceClient
-	cmd.RunE = func(cmd *cobra.Command, args []string) (err error) {
-		ctx := cmd.Context()
-		w := root.WorkspaceClient(ctx)
-
-		response, err := w.Settings.DeleteRestrictWorkspaceAdminsSetting(ctx, deleteRestrictWorkspaceAdminsSettingReq)
-		if err != nil {
-			return err
-		}
-		return cmdio.Render(ctx, response)
-	}
-
-	// Disable completions since they are not applicable.
-	// Can be overridden by manual implementation in `override.go`.
-	cmd.ValidArgsFunction = cobra.NoFileCompletions
-
-	// Apply optional overrides to this command.
-	for _, fn := range deleteRestrictWorkspaceAdminsSettingOverrides {
-		fn(cmd, &deleteRestrictWorkspaceAdminsSettingReq)
-	}
-
-	return cmd
-}
-
-func init() {
-	cmdOverrides = append(cmdOverrides, func(cmd *cobra.Command) {
-		cmd.AddCommand(newDeleteRestrictWorkspaceAdminsSetting())
-	})
-}
-
-// start get-default-namespace-setting command
-
-// Slice with functions to override default command behavior.
-// Functions can be added from the `init()` function in manually curated files in this directory.
-var getDefaultNamespaceSettingOverrides []func(
-	*cobra.Command,
-	*settings.GetDefaultNamespaceSettingRequest,
-)
-
-func newGetDefaultNamespaceSetting() *cobra.Command {
-	cmd := &cobra.Command{}
-
-	var getDefaultNamespaceSettingReq settings.GetDefaultNamespaceSettingRequest
-
-	// TODO: short flags
-
-	cmd.Flags().StringVar(&getDefaultNamespaceSettingReq.Etag, "etag", getDefaultNamespaceSettingReq.Etag, `etag used for versioning.`)
-
-	cmd.Use = "get-default-namespace-setting"
-	cmd.Short = `Get the default namespace setting.`
-	cmd.Long = `Get the default namespace setting.
-  
-  Gets the default namespace setting.`
-
-	cmd.Annotations = make(map[string]string)
-
-	cmd.Args = func(cmd *cobra.Command, args []string) error {
-		check := cobra.ExactArgs(0)
-		return check(cmd, args)
-	}
-
-	cmd.PreRunE = root.MustWorkspaceClient
-	cmd.RunE = func(cmd *cobra.Command, args []string) (err error) {
-		ctx := cmd.Context()
-		w := root.WorkspaceClient(ctx)
 
-		response, err := w.Settings.GetDefaultNamespaceSetting(ctx, getDefaultNamespaceSettingReq)
-		if err != nil {
-			return err
-		}
-		return cmdio.Render(ctx, response)
+		// This service is being previewed; hide from help output.
+		Hidden: true,
```

Comment:
2 things: 
* Latest OpenAPI spec is already public, so the next point will fix itself after the next Go SDK release.
* Settings (the parent service) is in Private Preview, but the actual settings are not. I think this change will hide the existing settings docs. 

I would remove this line manually.

Reply:
Yes. They were both set as Private as a workaround while docs made a fix. Everything is fixed now, but it was not included in the last release.

Reply:
Alright, flipping to unhide manually.